src/renderer/components/dock/upgrade-chart/view.tsx

```diff
@@ -58,7 +58,11 @@ export class NonInjectedUpgradeChart extends React.Component<UpgradeChartProps &
       ),
       reaction(
         () => this.release?.getRevision(),
-        () => this.reloadValues(),
+        () => {
+          if (this.release) {
+            this.props.upgradeChartTabStore.reloadValues(this.props.tab.id);
+          }
+        },
```

Comment:
It probably fixes the bug, but we need to understand why `this.release` can be undefined sometimes.